register-service-broker.sh

```
@@ -29,22 +29,14 @@ for SERVICE in $(echo "$SERVICES"); do
   # https://github.com/cloudfoundry/cli/issues/939
   cf disable-service-access "${ARGS[@]}"
 
-  # If SERVICE_ORGANIZATION_BLACKLIST, then expect service to be singular
+  if [ -n "${SERVICE_ORGANIZATION:-}" ] && [ -n "${SERVICE_ORGANIZATION_BLACKLIST:-}" ]; then
```

I'd move this above the for loop, we should fail fast.